doc/user/data/self_managed/aws_terraform_versions.yml

```diff
@@ -3,6 +3,11 @@ columns:
   - column: "Notable changes"
 
 rows:
+- "Terraform version": |
+    [v0.3.2](https://github.com/MaterializeInc/terraform-aws-materialize/releases/)
```

Comment:
the version is just a placeholder.  I'll update this to whatever release we go with.